cmd/registry/cmd/apply/testdata/lifecycle.yaml

```diff
@@ -9,28 +9,35 @@ data:
     - id: concept
       displayName: Concept
       description: Description of the business case and user needs for why an API should exist
+      url: ""
```

Comment:
All these fields are here because YAML export now includes default values for fields. The value of including empty strings is unclear, but zero-valued ints (displayOrder, below) seem important to include to allay (naive) concerns that they might be unset. The controls for this are not type-specific, so whatever we do for ints applies to strings, bools, etc.